cdk/autopilot/mapping/identifyMapping.yaml

```diff
@@ -0,0 +1,39 @@
+- destKey: Email
```

Comment:
This file is already under a `mapping` directory. We should not add redundant context with naming it again as `identifyMapping.yaml'.

Reply:
Agreed. This is to to serve as an example for the user that they can name it in anyway they deem fit

Reply:
@saikumarrs The point here is to give the user the flexibility to mention the path to the file where identify mapping yaml is present. They can give any name for that matter. So the useful thing about this is, say all mappings are same for all calls, you can have one default.yaml and point to it. 
Now on top of it, we can maybe add a layer of logic to autodeduce it (which it is not currently). But this being more direct would help the user to debug in case payload transformation is not working as expected.

Reply:
I totally get you. I'm just saying at least the examples we give should be the right ones.
`/mappings/identifyMapping.yaml` should just have been `/mappings/identify.yaml`. We can talk about the flexibility in the documentation.
It is understood that if you name an identify mapping file as `my_identify_mapping_file.yaml`, it needs to be specified in the config.

Reply:
@sanpj2292 To also answer this comment,
https://github.com/rudderlabs/rudder-transformer/pull/1007#discussion_r812612019

If the mapping files are simply named after the message types, `identify.yaml`, `track.yaml` etc in mappings directly, CDK should not expect the file paths in the config.
Isn't it intuitive that if the user names the files in a way CDK doesn't understand, it should be explicitly called out in the config?

Reply:
@saikumarrs 
Sure, we will implement the intuitive behavior for deducing the files